### ISSUE TITLE
"View an example" is a dead link

### ISSUE THREAD

**Issue by noyainrain** [noyainrain/meetling#1]
"View an example" on the start page is a dead link. Make it possible to create an example meeting.

**Comment by noyainrain** [comment]
Draft:

``` python
class Meetling:
    def create_example_meeting(self):
        """
        ..http:post:: /api/create-example-meeting

        Create a :ref:`Meeting` with an example agenda and return it.

        Useful to illustrate what Meetling is about.
        """
```

**Comment by noyainrain** [comment]
Fixed in a1f5fc9b8d170c8713578ffe11f6e7c4e1eab3f8 .
